.circleci/config.yml

```diff
@@ -56,9 +56,9 @@ jobs:
           name: Restore Yarn cache
           keys:
             # when lock file changes, use increasingly general patterns to restore cache
-            - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
-            - yarn-packages-v1-{{ .Branch }}-
-            - yarn-packages-v1-
+            - yarn-packages-v2-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - yarn-packages-v2-{{ .Branch }}-
+            - yarn-packages-v2-
```

Comment:
I'm changing the CircleCI cache prefix for our yarn packages just to make sure that we're not accidentally using any Node 10.x packages or something.